lsp-protocol.el

```diff
@@ -153,16 +153,20 @@ See `-let' for a description of the destructuring mechanism."
 
 ;; manually defined interfaces
 (lsp-interface (RpcRequestResponse (:params :id :method) nil))
+(lsp-interface (Command (:title :command) (:arguments)))
 
 (lsp-interface (css:ApplyCodeAction (:arguments) nil))
 
-(lsp-interface (eslint:StatusNotification (:state) nil))
-(lsp-interface (eslint:WorkspaceConfigurationReqHandler (:items) nil))
-(lsp-interface (eslint:WorkspaceConfiguration (:scopeUri) nil))
-(lsp-interface (eslint:OpenDoc (:url) nil))
+(lsp-interface (eslint:StatusNotification (:state) nil)
+               (eslint:WorkspaceConfigurationReqHandler (:items) nil)
+               (eslint:WorkspaceConfiguration (:scopeUri) nil)
+               (eslint:OpenDoc (:url) nil))
 
 (lsp-interface (haxe:ProcessStartNotification (:title) nil))
 
+(lsp-interface (pwsh:ApplyCodeActionEdits (:StartLineNumber :EndLineNumber :StartColumnNumber :EndColumnNumber :Text) nil)
```

Comment:
This should be `pwsh:ScopeRegion`.
https://github.com/PowerShell/PowerShellEditorServices/blob/3fcd0c651d50ec629b90c186c766af1872f81459/src/PowerShellEditorServices/Services/TextDocument/ScriptRegion.cs

Reply:
`ScriptRegion` you mean, right?
Thanks, I found the repo, but could not find these definitions.